spec/queries/get_activity_log_events_spec.rb

```diff
@@ -64,20 +64,19 @@ def create_audit_for_application_form(application_choice)
     end
 
     it 'defaults the query date range to the earliest scoped application form creation' do
```

Comment:
I'm not 100% convinced that this test is needed.

Reply:
Removed